src/main/java/com/emc/mongoose/storage/driver/pravega/io/ByteBufferSerializer.java

```diff
@@ -0,0 +1,44 @@
+package com.emc.mongoose.storage.driver.pravega.io;
+
+import io.pravega.client.stream.Serializer;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+
+public class ByteBufferSerializer
+implements Serializer<Integer>, Serializable {
```

Comment:
Looks to be a kind of bad idea. Let's use a deserializer which just returns the received byte buffer. Then we should just estimate the byte count in the buffer and forget it.